app/views/sales/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="sub-header">
+  <h3>All Time Sales for <%= @vendor.name %></h3>
+</div>
+<% total_sales = [] %>
```

Comment:
This template shares a great deal of code with the `current_month.html.erb` template. Is there a way that we could maybe DRY this out?